### DISCUSSION TITLE
Welcome to medusa-plugins Discussions!

### DISCUSSION THREAD

**Discussion by adrien2p** [adrien2p/medusa-plugins#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by guillmelo** [comment]
Hello, I am currently having to do a implementation of Microsoft Identity manager, is there a ETA on this feature?
Thanks !
